sunpy/coordinates/transformations.py

```diff
@@ -361,4 +356,18 @@ def hgs_to_hgs(from_coo, to_frame):
         return from_coo.transform_to(HCRS).transform_to(to_frame)
 
 
+@frame_transform_graph.transform(FunctionTransform, Heliocentric, Heliocentric)
+def hcc_to_hcc(from_coo, to_frame):
+    """
+    Convert from  Heliocentric to Heliocentric
+    """
+    if _observers_are_equal(from_coo.observer, to_frame.observer, string_ok=True):
+        return to_frame.realize_frame(from_coo._data)
+
+    hgs = from_coo.transform_to(HeliographicStonyhurst)
```

Comment:
Similar comment to above. Naming scheme should be a bit more explicit. `hgs`->`hgscoord`, `hcc`->`hcccoord`.